src/styles/_constants.scss

```diff
@@ -1,6 +1,20 @@
-$color1: #7a7265;
-$color2: #c0b7b1;
-$color3: #8e6e53;
-$color4: #c69c72;
-$color5: #433e3f;
+$body-background-color: #f5f9fc;
+$primary-text-color: #242527;
+$secondary-text-color: #738694;
+$divider-color: #d8e1e8;
+$background-color: #fff;
+$secondary-background-color: #f2f6f8;
+$header-background-color: #090809;
+$inverted-text-color: #fff;
+$inverted-background-color: rgb(234 240 245 / 10%);
+$card-header-text-color: #242527;
+$card-primary-text-color: #242527;
+$card-secondary-text-color: #696761;
+$primary-button-color: #1b9af0;
+$primary-button-text-color: #fff;
+$secondary-button-text-color: #1b9af0;
+$primary-font: roboto, sans-serif;
+$bold-font: 700;
+$regular-font: 400;
+$thin-font: 100;
```

Comment:
I don't think we will use anything this thin, it probably wouldn't be too readable. 400 and 700 should be enough for now.